app/test_main.py

```diff
@@ -1,3 +1,86 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
```

Comment:
Mmm… I’m not sure you need class to group tests as you have tests for only one function in this particular project. in the future, maybe. But not here. I would prefer to delete